sherpa-ncnn/csrc/recognizer.cc

```diff
@@ -85,7 +86,16 @@ void Recognizer::AcceptWaveform(float sample_rate, const float *input_buffer,
 
 void Recognizer::Decode() { decoder_->Decode(); }
 
-RecognitionResult Recognizer::GetResult() { return decoder_->GetResult(); }
+RecognitionResult Recognizer::GetResult() {
+  auto r = decoder_->GetResult();
+
+  float frame_shift_s = fbank_opts_->frame_opts.frame_shift_ms / 1000.;
+  for (auto &t : r.timestamps) {
+    t *= frame_shift_s;
```

Comment:
```suggestion
    t *= frame_shift_s * 4;
```

You can assume that the subsampling factor is 4.